machine/netberg/netberg_common_rangeley/busybox/patches/series

```diff
@@ -0,0 +1,3 @@
+onie-sys-eeprom-platform.patch
+mb_detect_c.patch
+mb_detect_h.patch
```

Comment:
All these "patches" logically belong in a single patch.

A patch should stand on its own, meaning it can be applied by itself and the code will still compile.  What you have here requires that all 3 patches be applied.